modules/dnn/test/test_onnx_conformance.cpp

```diff
@@ -311,6 +311,8 @@ static const TestCase testConformanceConfig[] = {
     {"test_gridsample_nearest", 2, 1},
     {"test_gridsample_reflection_padding", 2, 1},
     {"test_gridsample_zeros_padding", 2, 1},
+    {"test_group_normalization_epsilon", 3, 1},
+    {"test_group_normalization_example", 3, 1},
```

Comment:
https://github.com/opencv/opencv/blob/4.x/modules/dnn/test/test_onnx_conformance_layer_filter__openvino.inl.hpp should be updated to fix failed CI job: https://github.com/opencv/opencv/actions/runs/7337585388/job/19981699558?pr=24610